### DISCUSSION TITLE
RangeError: The value of "offset" is out of range. It must be an integer.

### DISCUSSION THREAD

**Discussion by candrea77** [zwave-js/zwave-js#6549] [Request Support / Investigate Issue] [answered]
### Checklist

- [X] I have read and followed the above instructions
- [X] I have checked the [troubleshooting section](https://zwave-js.github.io/node-zwave-js/#/troubleshooting/index) and my problem is **not** described there.

- [X] I have read the [changelog](https://github.com/zwave-js/node-zwave-js/blob/master/CHANGELOG.md) and my problem was **not** mentioned there or the fix **did not work**.

### Describe the issue

What is happening?
Today I run inside those things that's look like a bug to me.
I've got all zwave network completely freezed

What did you expect to happen instead?
Normal behaviour

At this stage I can't reproduce it.

Anything else we should know?
Never happen as far from today.
After docker stop / start , network start to work as expected.

### Software versions

zwavejs/zwave-js-ui:master (updated some days ago)

zwave-js-ui: 9.5.1.e4c1eb5
zwave-js: 12.4.0
home id: 4081278406
home hex: 0xf3435dc6

### Device information

Manufacturer: Aeotec
Model name:Zstick7 EU
Node ID: 001
FW: v7.17.2
SDK: v7.17.2

### Checklist

- [X] I [made sure](https://zwave-js.github.io/node-zwave-js/#/troubleshooting/index?id=rule-1-use-the-driver-log-luke) to provide a **driver log** on level **debug**.

- [X] The log includes a re-interview of the problematic device _(if applicable)_.

- [X] The log includes the problematic interaction with the device _(if applicable)_.

- [X] I provided the node ID of the problematic device _(if applicable)_.

### Upload Logfile

Here you are log, starting from today.
Problem stared at 2023-12-09T08:21:14.634Z
[zwavejs_current.log](https://github.com/zwave-js/node-zwave-js/files/13623793/zwavejs_current.log)

If you look at log , seems that controller report device with very High ID
Example : 
```
_2023-12-09T09:46:35.352Z CNTRLR   [Node 5791] is unknown - discarding received command...
2023-12-09T09:46:35.352Z DRIVER « [Node 5791] [REQ] [BridgeApplicationCommand]                       [discarded]
                                  │ target node: 409
                                  └─[ZWaveProtocolCC]
                                      payload: 0xa7_
```

**Comment by AlCalzone** [comment]
Seems that the controller was hanging and restarted itself somehow, but Z-Wave JS didn't understand.

**Comment by AlCalzone** [comment, marked as answer]
Tracking as a bug here: https://github.com/zwave-js/node-zwave-js/issues/6550
